src/sulogin-shell/sulogin-shell.c

```diff
@@ -88,7 +88,7 @@ static void print_mode(const char* mode) {
 }
 
 int main(int argc, char *argv[]) {
-        static const char* const sulogin_cmdline[] = {SULOGIN, NULL};
+        static const char* const sulogin_cmdline[] = {SULOGIN, "--force", NULL};
```

Comment:
hmm, so, i am not against this, but when i look at the man page of sulogin it has all the bold text regarding "start a root shell without a password" and stuff, and that really makes me a bit uncomfortable with doing this change this silently, without involving downstreams more. So, I think:

1. a NEWS section about this must be added, advertising this change for v236.
2. I think this should be configurable through a meson build-time configuration option, so that distros can chose this option or avoid it, and we just pick a default, but make it easy for downstreams to revert
3. I'd really like to hear something from @mbiebl from the Debian perspective on this. And @keszybz too.
4. maybe (and just maybe) we should also add a kernel cmdline option to override this value, i.e. "systemd.sulogin-force" or so.

Reply:
1 + 3: agreed.

2 + 4: if this needs to be more configurable, then I suggest that `systemd-sulogin-shell` changes its interface entirely: systemd-sulogin-shell passes its cmdline arguments to sulogin, and the name for the mode in the opening message can then be taken from an environment variable (should that message really be in sulogin-shell, though?). With that interface, rescue.service and emergency.service can separately decide whether to supply `--force` (and perhaps `--timeout`), as this change affects each target differently.